src/gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -497,8 +499,6 @@ properties:
       - Supraclavicular
       - Unknown
       - Not Reported
-    deprecated_enum:
-      - Cervical
```

Comment:
i didn't see this change (undeprecating?) in the ticket.

Reply:
Fixed; python + some packages were updated and the generating scripts are behaving weird, we'll probably have a few more of these 😞